src/lib.rs

```diff
@@ -245,6 +244,7 @@ pub fn to_pdf(
 /// ```
 /// # fn main() -> Result<(), Box<dyn std::error::Error>> {
 /// use std::collections::HashMap;
+/// use std::sync::Arc;
```

Comment:
can't select an unchanged line, but the `IMPORTANT` notice can be removed here (and abovE)